modules/openapi-generator/src/main/resources/C-libcurl/README.md.mustache

```diff
@@ -27,6 +27,16 @@ cd curl-7.61.0/
 make
 sudo make install
 ```
+Install the `curl 7.61` package with the following command on Linux.
```

Comment:
Copy and paste error: should be uncrustify 0.67 instead of curl 7.61

Reply:
Sorry, my mistake , will fix it.